v2/tools/generator/internal/config/always_matcher.go

```diff
@@ -1,36 +0,0 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT license.
- */
-
-package config
-
-// alwaysMatcher is a matcher that always matches
-type alwaysMatcher struct{}
```

Comment:
Why delete this?

Haven't you changed the behaviour for matching empty strings?

Reply:
It's never used. If the behavior had been changed CI would break and/or there would've been generated code changes. I tested w/ a breakpoint too and this path is never taken. See comment above about why.

Reply:
Having this conversation in the other comment thread so resolving this one